src/components/common/navBar.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+class NavBar extends Component {
+  render() {
+    return (
+      <div className="menu-header">
+        <img src="../assets/images/writing.png" className="logo" />
```

Comment:
also here `import Writing from '../assets/images/writing.png'` then use `src={Writing}` here.  Do that also everywhere you are using `src=""`